### ISSUE TITLE
Some more deprecated warnings

### ISSUE THREAD

**Issue by trickert76** [lae/ansible-role-proxmox#77]
This should be fixed for later Ansible version support.

```
TASK [lae.proxmox : Configure Ceph package source] ***********************************************************************************************************************
[DEPRECATION WARNING]: evaluating pve_ceph_enabled as a bare variable, this behaviour will go away and you might need to add |bool to the expression in the future. Also 
see CONDITIONAL_BARE_VARS configuration toggle.. This feature will be removed in version 2.12. Deprecation warnings can be disabled by setting deprecation_warnings=False
 in ansible.cfg.

TASK [lae.proxmox : Install Ceph packages] *******************************************************************************************************************************
[DEPRECATION WARNING]: evaluating pve_ceph_enabled as a bare variable, this behaviour will go away and you might need to add |bool to the expression in the future. Also 
see CONDITIONAL_BARE_VARS configuration toggle.. This feature will be removed in version 2.12. Deprecation warnings can be disabled by setting deprecation_warnings=False
 in ansible.cfg.

TASK [lae.proxmox : Install custom Ceph systemd service] *****************************************************************************************************************
[DEPRECATION WARNING]: evaluating pve_ceph_enabled as a bare variable, this behaviour will go away and you might need to add |bool to the expression in the future. Also 
see CONDITIONAL_BARE_VARS configuration toggle.. This feature will be removed in version 2.12. Deprecation warnings can be disabled by setting deprecation_warnings=False
 in ansible.cfg.

TASK [lae.proxmox : Enable Ceph] *****************************************************************************************************************************************
[DEPRECATION WARNING]: evaluating pve_ceph_enabled as a bare variable, this behaviour will go away and you might need to add |bool to the expression in the future. Also 
see CONDITIONAL_BARE_VARS configuration toggle.. This feature will be removed in version 2.12. Deprecation warnings can be disabled by setting deprecation_warnings=False
 in ansible.cfg.

TASK [lae.proxmox : Create initial Ceph config] **************************************************************************************************************************
[DEPRECATION WARNING]: evaluating pve_ceph_enabled as a bare variable, this behaviour will go away and you might need to add |bool to the expression in the future. Also 
see CONDITIONAL_BARE_VARS configuration toggle.. This feature will be removed in version 2.12. Deprecation warnings can be disabled by setting deprecation_warnings=False
 in ansible.cfg.

TASK [lae.proxmox : Create initial Ceph monitor] *************************************************************************************************************************
[DEPRECATION WARNING]: evaluating pve_ceph_enabled as a bare variable, this behaviour will go away and you might need to add |bool to the expression in the future. Also 
see CONDITIONAL_BARE_VARS configuration toggle.. This feature will be removed in version 2.12. Deprecation warnings can be disabled by setting deprecation_warnings=False
 in ansible.cfg.

TASK [lae.proxmox : Fail if initial monitor creation failed] *************************************************************************************************************
[DEPRECATION WARNING]: evaluating pve_ceph_enabled as a bare variable, this behaviour will go away and you might need to add |bool to the expression in the future. Also 
see CONDITIONAL_BARE_VARS configuration toggle.. This feature will be removed in version 2.12. Deprecation warnings can be disabled by setting deprecation_warnings=False
 in ansible.cfg.

TASK [lae.proxmox : Create additional Ceph monitors] *********************************************************************************************************************
[DEPRECATION WARNING]: evaluating pve_ceph_enabled as a bare variable, this behaviour will go away and you might need to add |bool to the expression in the future. Also 
see CONDITIONAL_BARE_VARS configuration toggle.. This feature will be removed in version 2.12. Deprecation warnings can be disabled by setting deprecation_warnings=False
 in ansible.cfg.

TASK [lae.proxmox : List Ceph CRUSH rules] *******************************************************************************************************************************
[DEPRECATION WARNING]: evaluating pve_ceph_enabled as a bare variable, this behaviour will go away and you might need to add |bool to the expression in the future. Also 
see CONDITIONAL_BARE_VARS configuration toggle.. This feature will be removed in version 2.12. Deprecation warnings can be disabled by setting deprecation_warnings=False
 in ansible.cfg.

TASK [lae.proxmox : List Ceph Pools] *************************************************************************************************************************************
[DEPRECATION WARNING]: evaluating pve_ceph_enabled as a bare variable, this behaviour will go away and you might need to add |bool to the expression in the future. Also 
see CONDITIONAL_BARE_VARS configuration toggle.. This feature will be removed in version 2.12. Deprecation warnings can be disabled by setting deprecation_warnings=False
 in ansible.cfg.
TASK [lae.proxmox : Create Ceph MDS servers] *****************************************************************************************************************************
[DEPRECATION WARNING]: evaluating pve_ceph_enabled as a bare variable, this behaviour will go away and you might need to add |bool to the expression in the future. Also 
see CONDITIONAL_BARE_VARS configuration toggle.. This feature will be removed in version 2.12. Deprecation warnings can be disabled by setting deprecation_warnings=False
 in ansible.cfg.

TASK [lae.proxmox : Wait for standby MDS] ********************************************************************************************************************************
[DEPRECATION WARNING]: evaluating pve_ceph_enabled as a bare variable, this behaviour will go away and you might need to add |bool to the expression in the future. Also 
see CONDITIONAL_BARE_VARS configuration toggle.. This feature will be removed in version 2.12. Deprecation warnings can be disabled by setting deprecation_warnings=False
 in ansible.cfg.

TASK [lae.proxmox : List Ceph Filesystems] *******************************************************************************************************************************
[DEPRECATION WARNING]: evaluating pve_ceph_enabled as a bare variable, this behaviour will go away and you might need to add |bool to the expression in the future. Also 
see CONDITIONAL_BARE_VARS configuration toggle.. This feature will be removed in version 2.12. Deprecation warnings can be disabled by setting deprecation_warnings=False
 in ansible.cfg.
```

**Comment by lae** [comment]
@trickert76 how are you reproducing this? are you perhaps on an older version of the role?

I'm not seeing the deprecation warnings in the latest travis run for the development version of ansible: https://travis-ci.org/lae/ansible-role-proxmox/jobs/600047875#L1709

**Comment by lae** [comment]
In fact I think you submitted the fix for this a few months ago: a50bbc9d5cd8c4d044f2533f1388c70449b6d878

Closing this since as far as I can tell it appears to be resolved, but if not, let me know.

**Comment by trickert76** [comment]
Hmm, I've really checked it before. I'm on your develop branch and I've made a ansible-galaxy force update. I've run the playbook twice to verify. I need to check that again - because in Travis the task names differ from my log. Sorry.
